packages/web/src/utils/regex.ts

```diff
@@ -1 +1,3 @@
-export const REGEX_NUMBER_FORMAT = /\.00|(\.\d*?)0+$/g;
\ No newline at end of file
+export const REGEX_NUMBER_FORMAT = /\.00|(\.\d*?)0+$/g;
+
+export const cutDecimalNumberWithoutRounding = (digits: number) => new RegExp("^-?\\d+(?:\.\\d{0," + (digits || -1) + "})?");
```

Comment:
If you want to express discarding decimals, it seems better to use `roundDown` than `cut`.

Reply:
so i will change it to `roundDownDecimalNumber`